### DISCUSSION TITLE
Problem while doing ssh on a server without a password

### DISCUSSION THREAD

**Discussion by nyxonman** [kingToolbox/WindTerm#504] [Q&A]
I have recently tried windterm and I liked it very much. However, I am facing an issue while trying to perform ssh to a server that has no password set. There is a user@ipaddr. When I try to sign in, it freezes and does not work. I have tried the same with putty or windows terminal and it is working fine. 

It is stuck in this login page
![image](https://user-images.githubusercontent.com/41434353/153831306-d3b6e52e-ca77-4385-b9f8-d0591f6a682f.png)

**Comment by kingToolbox** [comment]
This looks like a bug, I will investigate this issue and update here if there are any progress. Please stay tuned, thank you.

**Reply by nyxonman** [reply]
Any update on this problem? Also i am also facing to ssh on devices

![image](https://user-images.githubusercontent.com/41434353/157701107-a5697797-fec0-4de3-8baf-c8e851919def.png)

![image](https://user-images.githubusercontent.com/41434353/157701167-8fd3eb7a-778f-43aa-a6ce-045afeb92fc5.png)
 
And after pressing continue, it fails and WindTerm freezes

![image](https://user-images.githubusercontent.com/41434353/157701867-007af35c-0944-4a65-a23f-7adcdf93306c.png)

**Reply by mizzuri** [reply]
@nyxonman, can you try again with the following settings? (Assuming you're using a public-key authentication.)
Also make sure you click on "Clear" for "Saved automatic authentication".

![image](https://user-images.githubusercontent.com/1288437/158618626-7cee7115-670f-49ee-93a4-0864bdcdd2aa.png)

**Reply by kingToolbox** [reply]
I am happy to inform you that [WindTerm_2.4.0](https://github.com/kingToolbox/WindTerm/releases/tag/2.4.0) has been released and this bug has been fixed. Please download and check it, thank you.

**Reply by nyxonman** [reply]
Thanks it works better now.

**Reply by Akira657** [reply]
I am using WindTerm_2.4.1_Windows_Portable_x86_64, the bug is still exist.
